packages/@aws-cdk/aws-ecs-patterns/test/fargate/integ.scheduled-fargate-task.lit.expected.json

```diff
@@ -232,6 +232,7 @@
                   ]
                 }
               },
+              "PlatformVersion": "LATEST",
```

Comment:
Are the existing tests needing to be changed because of the default we're setting of `LATEST`?

Reply:
yes.
I wasn't aware of the need for this change either, but the test failed so I fixed it.

Reply:
needing to update an existing is a sign that the change might be considered breaking / can introduce a regression. what happens if we don't set it to `LATEST` explicitly? that should represent the current behaviour right?